app/components/Search/Search.css

```diff
@@ -140,12 +141,33 @@
 .resultDate {
   color: var(--lego-font-color);
   margin-left: auto;
+  white-space: nowrap;
```

Comment:
Make sure to check stuff like this on Safari and its iOS developer mode. Should be good but those have acted up in the past, and making sure the site looks good on iPhone is pretty important.